### DISCUSSION TITLE
Connect to all schemas in the database?

### DISCUSSION THREAD

**Discussion by swdmike** [developmentseed/tipg#124] [Q&A]
It would be helpful if I could set TIPG_DB_SCHEMAS='["*"]' to connect to all schemas in my database. In my work environment, schemas are frequently created based on project requirements, with each project having one or more schemas. I want to be able to see all spatial data within TIPG without constantly modifying its configuration.

Thanks!

**Comment by vincentsarago** [comment]
@swdmike I don't think it's currently possible but you could customize this https://github.com/developmentseed/tipg/blob/fdb6ec65f6ca3aa5b773a9943aa43513d3f2cf8e/tipg/database.py#L23-L67 to first list all the schemas and then construct the search path https://github.com/developmentseed/tipg/blob/fdb6ec65f6ca3aa5b773a9943aa43513d3f2cf8e/tipg/database.py#L49

**Reply by swdmike** [reply]
Thanks, it's done.

What if I create a new schema, how to add it by code if I donot want to restart tipg.
